src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/VisualBasicCodeFixVerifier`2+Test.cs

```diff
@@ -28,7 +27,7 @@ public Test()
 
                 SolutionTransforms.Add((solution, projectId) =>
                 {
-                    var parseOptions = (VisualBasicParseOptions)solution.GetRequiredProject(projectId).ParseOptions!;
+                    var parseOptions = (VisualBasicParseOptions)solution.GetProject(projectId)?.ParseOptions!;
```

Comment:
❕ `GetRequiredProject` is the desired behavior here.

```suggestion
                    var parseOptions = (VisualBasicParseOptions)solution.GetProject(projectId)!.ParseOptions!;
```